scripts/mk_all.sh

```diff
@@ -7,8 +7,18 @@ for d in $(find $DIR/../src/ -type d)
 # For every subfolder of src/ (including src/ itself)
 do
   cd "$d" # cd to that directory
+  echo /- automatically generated file importing all files in this directory and subdirectories. -/ > all.lean
   find . -maxdepth 1 -mindepth 1 -name 'all.lean' -prune -o -name '*.lean' -print -o -type d -print |
   # find all files with .lean extension (except all.lean) and all subdirectories
-  sed 's/$/\.all/; s/\.lean\.all//; s/^\.\//       \./; 1s/^      /\/- automatically generated file importing all files in this directory and subdirectories. -\/\nimport/' > all.lean
+  sed 's/$/\.all/; s/\.lean\.all//; s/^\.\//       \./; 1s/^      /import/' >> all.lean
   # Now modify this output so that Lean can parse it. Changes `./foo.lean` to `.foo` and `foodir` to `.foodir.all`. Also adds indentation, a comment and `import`. Write this to `all.lean`
 done
+
+cd $DIR/../src/
+cat <<EOT >> sanity_check_mathlib.lean
```

Comment:
```suggestion
cat <<EOT > sanity_check_mathlib.lean
```
Let’s just overwrite the sanity_check_mathlib file.